search/search_quality/assessment_tool/main_model.cpp

```diff
@@ -64,24 +72,53 @@ void MainModel::OnSampleSelected(int index)
     params.m_suggestsEnabled = false;
     params.SetPosition(latLon.lat, latLon.lon);
 
-    auto const timestamp = ++m_queryTimestamp;
+    auto const timestamp = m_queryTimestamp;
     m_numShownResults = 0;
 
-    params.m_onResults = [this, timestamp](search::Results const & results) {
-      GetPlatform().RunOnGuiThread([this, timestamp, results]() { OnResults(timestamp, results); });
+    params.m_onResults = [this, sample, timestamp](search::Results const & results) {
+      vector<search::Sample::Result::Relevance> relevances;
+      if (results.IsEndedNormal())
+      {
+        search::Matcher matcher(m_index);
+
+        vector<search::Result> const actual(results.begin(), results.end());
+        std::vector<size_t> goldenMatching;
```

Comment:
Ok. If so add please std prefixes where it's necessary. For example at the string above.